src/lib/__tests__/InstantSearch-integration-test.ts

```diff
@@ -0,0 +1,46 @@
+import { getByText, fireEvent } from '@testing-library/dom';
```

Comment:
I think this file should just be called `InstantSearch-integration`, it's already in a tests folder

Reply:
I cannot find it...?

Reply:
I think he meant to rename `InstantSearch-integration-test.ts` to `InstantSearch-integration.ts`

Reply:
Ah right. I misunderstood. I was looking for `InstantSearch-integration.ts` file.
But the thing is, all our test files end with `-test.*` in `__tests__`.

Reply:
ah I didn't notice that, fine for me then 👍